ui/lib/pki/addon/engine.js

```diff
@@ -0,0 +1,29 @@
+import Engine from 'ember-engines/engine';
+import loadInitializers from 'ember-load-initializers';
+import Resolver from './resolver';
+import config from './config/environment';
+
+const { modulePrefix } = config;
+/* eslint-disable ember/avoid-leaking-state-in-ember-objects */
+const Eng = Engine.extend({
```

Comment:
Should we update this syntax to `extends Engine` - that way we can also remove the `eslint-disable` comment? 

Sample below from [ember engines quickstart](https://ember-engines.com/docs/quickstart) which is also what Chelsea's used in the Vault Insights engine [PR ](https://github.com/hashicorp/cloud-ui/pull/2936/files#diff-5ddf30e3d11c22cf19c8c1c488986e7c4363f85dec2aac48698c7a95b30acf2aR1)

```
// addon/engine.js
import Engine from '@ember/engine';

import loadInitializers from 'ember-load-initializers';
import Resolver from 'ember-resolver';

import config from './config/environment';

const { modulePrefix } = config;

export default class PkiEngine extends Engine {
  modulePrefix = modulePrefix;
  Resolver = Resolver;
}

loadInitializers(PkiEngine, modulePrefix);

Reply:
Thank you for pointing this out!  I'll update.